src/main/java/io/github/soc/directories/Util.java

```diff
@@ -111,34 +111,62 @@ static String linuxExecutableDir(String homeDir, String dataDir) {
   }
 
   static String[] getWinDirs(String... guids) {
+
+    // See https://www.oracle.com/technetwork/java/javase/8u231-relnotes-5592812.html#JDK-8221858
+    // If a security manager is set, ProcessBuilder should safely escape things.
+    final boolean doubleEscapeQuotes = System.getSecurityManager() == null;
+    final String[] initialResult = getWinDirs(doubleEscapeQuotes, guids);
+
+    boolean hasNonNull = initialResult.length == 0;
+    for (String s : initialResult) {
+      if (s != null) {
+        hasNonNull = true;
```

Comment:
Just straight `return initialResult;` here.

Reply:
Of course, thanks! Should be fixed by the latest commit.